src/components/Agent/AgentTable.tsx

```diff
@@ -21,19 +22,21 @@ const getColumns = () => [
 
 const AgentTable: React.FC = () => {
   const [hosts] = useK8sWatchResource<K8sResourceCommon[]>({
-    kind: `agent-install.openshift.io~v1beta1~Agent`,
+    kind: `agent-install.openshift.io~v1beta1~Agent`, // TODO(mlibra): compose from a model
```

Comment:
Console is getting away from models which are just a copy-paste from k8s resource definition. 

we should only need `apiGroup~apiVersion~Kind` as ID.
so instead of creating a model, we need to only have a string constant `agent-install.openshift.io~v1beta1~Agent` + the same for other resources

Reply:
ok, thanks. Removing the TODO comment here